x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/analytics_table.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('AnalyticsTable', () => {
     },
     {
       key: 'another search',
-      tags: ['tagB'],
+      tags: ['tagB', 'tagC'],
```

Comment:
I'm not a fan of changing the `AnalyticsTable` test file to test something that changed in `InlineTagsList`. In theory the changes made in this PR should be covered by the unit tests of `InlineTagsList` itself, `AnalyticsTable` should only know that it renders a `InlineTagsList` regardless of the content of `InlineTagsList`. This is _generally_ why I advocate for using `shallow()` in tests instead of `mount()` - it promotes more modular & separate unit tests.

Sorry, back to the nuts and bolts - I'd like request this file be reverted to its original content without changes. If we lose coverage on `InlineTagsList` because of that, that means we need to update `inline_tags_list.test.tsx`, not `analytics_tables.test.tsx`.

Reply:
Ah, wait, I think I see, sorry - this test needs to be updated anyway on line 51 (`tagA`->`1 tag`). I'm still not fully sold on the necessity of adding a `expect(tableContent).toContain('2 tags');` check but on second thought it's likely fine and isn't covering the conditional I originally thought it was.

Going to collapse this comment/change request, feel free to disregard it